Sprint-3/reading-list/script.js

```diff
@@ -21,3 +21,24 @@ const books = [
   },
 ];
 
+
+const list = document.getElementById('reading-list');
+
+function readingList(books){
+  books.forEach((book) => {
+    const listItem = document.createElement('li');
+    listItem.className = 'list-item';
+    list.appendChild(listItem);
```

Comment:
I would probably do this as the very _last_ thing after you fill in all of the `listItem` children, rather than near the top of this function - can you think why?